docs/core/v3.0.rst

```diff
@@ -0,0 +1,1763 @@
+.. This file is in restructured text format: http://docutils.sourceforge.net/rst.html
+.. _zarr-core-specification-v3.0:
+
+======================================
+ Zarr core specification (version 3.0)
+======================================
+-------------------------------
+ Editor's draft 25 May 2022
+-------------------------------
+
+Specification URI:
+    https://purl.org/zarr/spec/core/3.0
+    
+Editors:
+    * Alistair Miles (`@alimanfoo <https://github.com/alimanfoo>`_), Wellcome Sanger Institute
+    * Jonathan Striebel (`@jstriebel <https://github.com/jstriebel>`_), Scalable Minds
+
+Issue tracking:
+    `GitHub issues <https://github.com/zarr-developers/zarr-specs/labels/core-spec-v3.0>`_
+
+Suggest an edit for this spec:
+    `GitHub editor <https://github.com/zarr-developers/zarr-specs/blob/core-protocol-v3.0-dev/docs/core/v3.0.rst>`_
+
+Copyright 2019-Present `Zarr core development team
+<https://github.com/orgs/zarr-developers/teams/core-devs>`_. This work
+is licensed under a `Creative Commons Attribution 3.0 Unported License
+<https://creativecommons.org/licenses/by/3.0/>`_.
+
+----
+
+
+Abstract
+========
+
+This specification defines the Zarr format for N-dimensional typed arrays.
+
+
+Status of this document
+=======================
+
+This document is a **Work in Progress**. It may be updated, replaced
+or obsoleted by other documents at any time.
+
+Comments, questions or contributions to this document are very
+welcome. Comments and questions should be raised via `GitHub issues
+<https://github.com/zarr-developers/zarr-specs/labels/core-spec-v3.0>`_.
```

Comment:
```suggestion
```

I propose to remove this part of the front matter, as we want to solicit reviews and suggestions via the current PR, not via issues.

Reply:
Idea: We could host this PR draft via a readthedocs PR preview on Github Actions.

Reply:
That's basically what [this is]( https://zarr-specs.readthedocs.io/en/latest/ )

Reply:
Oh awesome! Sorry, I was on mobile and didn't realize.

Reply:
Actually this PR is a bit of a special case, because the spec got merged prior to the ZEP process being defined, hence the live readthedocs site effectively shows a preview of what's in this PR. But for future proposals going forward via the regular process, where the new spec content is only in a PR, it would be very cool to be able to have a preview. I see [readthedocs supports previewing documentation from pull requests](https://docs.readthedocs.io/en/stable/pull-requests.html), we should set that up.

Reply:
The box to enable RTD PR builds is already checked. Not sure why we don't see PR previews here. Maybe there is something else we are missing?

<details>
<summary>Screenshot:</summary>

<img width="1792" alt="Screen Shot 2022-07-25 at 9 43 28 PM" src="https://user-images.githubusercontent.com/3019665/180924912-bb51fdbb-d603-4ab8-89ff-9d21d4c0d6e7.png">

</details>

Reply:
We were missing the "Pull requests" permission on the GitHub webhook for RTD. Enabled that and then closed/reopen the PR to build the preview. Seems to be working 🎉

Reply:
Excellent! Thanks for doing this. :)

Reply:
Updated accordingly in #168. Marking as resolved here.